### ISSUE TITLE
Encoding of register 43416 "Compressor starts EB100-EP14"

### ISSUE THREAD

**Issue by samboman** [bebben88/NibePi#8]
When selecting the register 43416, I constantly get notifications about a corrupt value. 
"Logg - Korrupt data (-46821) från register 43416"

I can read the value from MQTT nibe/modbus/43416 as -46821.
Checking the menu at my Nibe F470 gives: 46821 compressor-starts (positive value)

Maybe its something wrong with the encoding?

I guess I have some info here in the log...

```Value: "40033: 21.9 °C"
Serial: [92,0,32,107,0,75]
OK: [92,0,32,107,0,75]
Serial: [92,0,32,105,0,73]
OK: [92,0,32,105,0,73]
Serial: [6]
Serial: [92,0,32,104,80,255,255,0]
Serial: [0,255,255,0,0,255,255,0]
Serial: [0,255,255,0,0,255,255,0]
Serial: [0,255,255,0,0,255,255,0]
Serial: [0,255,255,0,0,255,255,0]
Serial: [0,255,255,0,0,255,255,0]
Serial: [0,255,255,0,0,255,255,0]
Serial: [0,255,255,0,0,255,255,0]
Serial: [0,255,255,0,0,255,255,0]
Serial: [0,255,255,0,0,255,255,0]
Serial: [0,255,255,0,0,24]
OK: [92,0,32,104,80,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,24]
Serial: [92,0,32,106,6,159,183,8]
Serial: [0,255,255,108]
OK: [92,0,32,106,6,159,183,8,0,255,255,108]
Value: "47007: 8 "
Serial: [92,0,32,107,0,75]
OK: [92,0,32,107,0,75]
Serial: [92,0,32,105,0,73]
OK: [92,0,32,105,0,73]
Serial: [6]
Serial: [92,0,32,106,6,89,156,211]
Serial: [0,24,0,66]
OK: [92,0,32,106,6,89,156,211,0,24,0,66]
Value: "40025: 21.1 °C"
Serial: [92,0,32,107,0,75]
OK: [92,0,32,107,0,75]
Serial: [92,0,32,105,0,73]
OK: [92,0,32,105,0,73]
Serial: [6]
Serial: [92,0,32,104,80,255,255,0]
Serial: [0,255,255,0,0,255,255,0]
Serial: [0,255,255,0,0,255,255,0]
Serial: [0,255,255,0,0,255,255,0]
Serial: [0,255,255,0,0,255,255,0]
Serial: [0,255,255,0,0,255,255,0]
Serial: [0,255,255,0,0,255,255,0]
Serial: [0,255,255,0,0,255,255,0]
Serial: [0,255,255,0,0,255,255,0]
Serial: [0,255,255,0,0,255,255,0]
Serial: [0,255,255,0,0,24]
OK: [92,0,32,104,80,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,255,255,0,0,24]
Serial: [92,0,32,106,6,152,169,229]
Serial: [182,0,0,46]
OK: [92,0,32,106,6,152,169,229,182,0,0,46]
Value: "43416: -46821 "

**Comment by bebben88** [comment]
Decoding s32 registers with current version is broken. It will however be resolved in version 1.1. (43416 is a s32 register).

**Comment by samboman** [comment]
Okay, I see!
How do I see which is s32 registers?
